vanilla-app/package.json

```diff
@@ -14,17 +14,16 @@
     "vite": "4.5.0"
   },
   "dependencies": {
-    "@justeat/pie-design-tokens": "5.9.0",
```

Comment:
Was this okay to remove then?

Reply:
yepp, it's part of pie-css and not used anyway 〽️